docs/user-guide/howto.rst

```diff
@@ -236,6 +236,53 @@ to your code:
 
 .. accordion-header::
     :id: collapseHowToSixteen
+    :title: Add PHASE information to your data
+
+For doing pulsar analysis, you must compute the phase associated
+to each event and then create a new `EventList` and a new `Observation`.
+Modifying the `EventList` of an `Observation` in-place is prohibited because of the
+underlying lazy loading implemented in reading observations. Code for computing phases is
+NOT provided within gammapy, and you must use an external s/w like PINT or TEMPO2. For brevity, 
+this code example shows the only technical implementation using a dummy phase column.
+
+.. testcode::
+
+    import numpy as np
+    from gammapy.data import DataStore, Observation, EventList
+
+    # read the observation
+    datastore = DataStore.from_dir("$GAMMAPY_DATA/hess-dl3-dr1/")
+    obs = datastore.obs(23523)
+
+    # use the phase information - dummy in this example
+    phase = np.random.random(len(obs.events.table))
+
+    # create a new `EventList`
+    table = obs.events.table
+    table["PHASE"] = phase
+    events_new = EventList(table)
+
+    # create a new observation
+    o2 = Observation(obs_id=obs.obs_id,
```

Comment:
I think this needs simplification. What about adding `Observation.copy(in_memory=True)`?